_data-prepper/pipelines/configuration/processors/convert_entry_type.md

```diff
@@ -14,10 +14,22 @@ The `convert_entry_type` processor converts a value type associated with the spe
 
 You can configure the `convert_entry_type` processor with the following options.
 
+<!--
+This table is autogenerated. Do not edit it.
+- name: convert_entry_type
+- pluginType: processor
+- source: https://github.com/opensearch-project/data-prepper/blob/7d15115c281687aab50e5c471fd210cb1ef90fc5/data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/ConvertEntryTypeProcessorConfig.java
+-->
+
 | Option | Required | Description |
 | :--- | :--- | :--- |
-| `key`| Yes | Keys whose value needs to be converted to a different type. |
-| `type` | No | Target type for the key-value pair. Possible values are `integer`, `double`, `string`, and `Boolean`. Default value is `integer`. |
+| `key`| Yes | Key whose value needs to be converted to a different type. |
+| `keys`| Yes | Keys whose value needs to be converted to a different type. |
+| `type` | No | Target type for the key-value pair. Possible values are `integer`, `long`, `double`, `big_decimal`, `string`, and `boolean`. Default value is `integer`. |
+| `null_values` | No | String representation of what constitutes a `null` value. If the field value equals one of these strings, then the value is considered `null` and is converted to `null`. |
+| `scale` | No | Modifies the scale when used to convert to a `big_decimal`. The default value is `0`. |
```

Comment:
Is this saying that scale is only modified when the option is converted to `big_decimal`?

Reply:
I just pushed a new commit that attempts to correct the wording.

We are modifying the scale of the `big_decimal` value itself. This field is only applicable for `big_decimal`.